lib/notifications/display.dart

```diff
@@ -139,6 +142,22 @@ class NotificationDisplayManager {
         // TODO this doesn't set the Intent flags we set in zulip-mobile; is that OK?
         //   (This is a legacy of `flutter_local_notifications`.)
         ),
+      autoCancel: true,
```

Comment:
```
notif: Create summary notification
```

I see that zulip-mobile also does this (`setAutoCancel(true)`). Is this change coherent with the rest of what this commit is doing to create a summary notification? Or could it stand separately as its own commit? The answer will help me understand the user-facing changes that we're making, and how they're related.

Reply:
Moved this to a separate commit — it's not really needed for Group summary notifications, but was added just to make the UX better, and match the zulip-mobile implementation.